src/lib/components/LayeredMap.js

```diff
@@ -47,20 +49,28 @@ class LayeredMap extends Component {
                             </Overlay>
                         ))}
                     </LayersControl>
-                    <FeatureGroup>
-                        <DrawControls
-                            lineCoords={(coords) => this.props.setProps({'line_points': coords})}
-                            markerCoords={(coords) => this.props.setProps({'marker_point': coords})}
-                            polygonCoords={(coords) => this.props.setProps({'polygon_points': coords})} />
-                    </FeatureGroup>
+                    { showDrawControls  && (
+                        <FeatureGroup>
+                            <DrawControls
+                                showMarker={show_marker}
+                                showPolygon={show_polygon}
+                                showPolyline={show_polyline}
+                                lineCoords={(coords) => setProps({'line_points': coords})}
+                                markerCoords={(coords) => setProps({'marker_point': coords})}
+                                polygonCoords={(coords) => setProps({'polygon_points': coords})} />
+                                </FeatureGroup>
+                    )}
                 </Map>
         );
     }
 }
 
 LayeredMap.defaultProps = {
     height: 800,
-    hillShading: true
+    hillShading: true,
+    show_marker: false,
+    show_polygon: false,
+    show_polyline: false
```

Comment:
Should the props be prefixed with e.g. draw_toolbar instead of show, just to avoid any chance of confusion with layer data input?